packages/react-cookie/src/BrowserCookieManager.ts

```diff
@@ -0,0 +1,65 @@
+import cookie from 'cookie';
+import {hasDocumentCookie} from './utilities';
+import {CookieValue, Cookie} from './types';
+
+export {CookieContext} from './context';
+
+export class BrowserCookieManager {
```

Comment:
I guess this is consistent with the rest of Quilt, but I miss the lint rule for "this isn't really a class".

Reply:
Yeah this could easily just be a constant object exported directly since it doesn't have any local state. Doesn't matter too much.